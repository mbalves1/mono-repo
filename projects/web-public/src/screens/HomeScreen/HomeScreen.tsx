import React from "react";
import { sum } from "@monorepo/utils/math/sum";
import { Text } from "@monorepo/design-system/components/Text";

export function HomeScreen(){ 
  return (
    <main>
      <Text tag="h1"> Home </Text>
      <Text tag="p">Importanto modulo local: @monorepo/utils/math/sum(2,5) = {sum(2, 5)}</Text>
    </main>
  )
}
