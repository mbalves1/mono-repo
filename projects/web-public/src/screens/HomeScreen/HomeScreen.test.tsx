import { customRender } from '@monorepo/test-commons/react-testing-library';
import { HomeScreen } from './HomeScreen';

function CustomWrapper({children}) {
  console.log('cutsom wrapper');
  return <section>{children}</section>
}
const render = customRender(CustomWrapper);

describe('<HomeScreen />', () => {
  it('renders the page', () => {
    const {container} = render(<HomeScreen />)

    expect(container).toMatchSnapshot();
  })
})
