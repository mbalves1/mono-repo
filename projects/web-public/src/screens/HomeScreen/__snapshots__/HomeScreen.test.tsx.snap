// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders the page 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <main>
    <h1
      class="c0"
    >
       Home 
    </h1>
    <p
      class="c0"
    >
      Importanto modulo local: @monorepo/utils/math/sum(2,5) = 
      7
    </p>
  </main>
</div>
`;
